### PR TITLE
CONTENT-5669 - Build Modern Blog with GatsbyJS and MongoDB Post 

### PR DESCRIPTION
**Pull request by kukicado**
Updated URL: http://docs-devhub-staging.s3-website-us-east-1.amazonaws.com/03a345d/devhub/docsworker-xlarge/gatsby-mongodb/how-to/gatsby-modern-blog

### CODE REVIEW

File: source/how-to/gatsby-modern-blog.txt [hunk 1]
```diff
@@ -0,0 +1,880 @@
...
+
+.. ---------------------------------------------------------------
+.. ARTICLE METADATA FIELDS (TO POPULATE FILTERS AND ARTICLE PAGE)
+.. ---------------------------------------------------------------
+
+.. include:: /includes/authors/kukic-ado.rst
+
+.. atf-image:: /images/atf-images/generic/purple.png
+
+.. pubdate:: 2020-03-15
```
Comment (kenwalger) [inline source/how-to/gatsby-modern-blog.txt]: Please change to 2020-03-24

File: source/how-to/gatsby-modern-blog.txt [hunk 2]
```diff
@@ -0,0 +1,880 @@
...
+
+.. summary::
+
+   Putting It All Together
+   -----------------------
+
+   In this tutorial, I showed you how to build a modern blog with GatsbyJS. We used multiple data sources, including a remote `MongoDB Atlas <https://mongodb.com/cloud/atlas>`__ database and local markdown files, to generate a static blog. We took a brief tour of GraphQL and how it enhances our development experience by consolidating all of our data sources into a single API that we can query both at build and run time. I hope you learned something new, if you have any questions feel free to ask in our `MongoDB community forums <https://community.mongodb.com/>`__.
+
+   If you want to get the code for this tutorial, you can clone it from this `GitHub repo <https://github.com/kukicado/gatsby-mongodb-article>`__. The sample books dataset can also be found `here <https://github.com/ozlerhakan/mongodb-json-files>`__. Try `MongoDB Atlas <https://mongodb.com/cloud/atlas>`__ to make it easy to manage and scale your MongoDB database and use code **ADO200** for a $200 credit.
```
Comment (kenwalger) [inline source/how-to/gatsby-modern-blog.txt]: Make this a blockquote directive so the CTA stands out in the text.

**Comment by kukicado** [comment]
Changes made and preview URL updated. We are good to go for 3/24 on this post.

**Comment by MaBeuLux88** [comment]
Good to publish on March 24th

**Comment by kukicado** [comment]
🙌

**Pull request merged**